langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -64,10 +79,21 @@ public isolated function isNaN(float x) returns boolean = @java:Method {
 
 # Returns the sum of zero or more float values.
 #
-# Result is NaN if any arg is NaN
+# ```ballerina
+# // Take the sum of multiple float numbers.
+# float total = float:sum(1.2, 2.3, 3.4);
+#
+# // Take the sum of existing float number list.
+# float[] scores = [11.1, 22.2, 33.3];
+# float totalScore = float:sum(...scores);
+#
+# // Take the sum of multiple float numbers with an existing float number.
+# float value = 21.2;
+# float summation = value.sum(10.5, 21, 32.4);
```

Comment:
This example is added because Ballerina has this functionality. But this may not a good usage.
Shall I remove this example?.
applicable for `min, max` as well.

Reply:
This isn't just with an existing float variable. Even `21.2.sum(10.5, 21, 32.4)` works. I think it's OK to add an example, but let's change the description to say you can also use a method call expression.

Reply:
I agree with `min, max, sum`. But it is not helpful with `atan2` ryt?

Reply:
+1 to have for just min, max, and sum.

Reply:
Updated